README.md

```diff
@@ -411,11 +411,9 @@ okta:
       username: null
       password: null
     authorizationMode: "PrivateKey"
-    clientId: "{clientId}"
-    scopes:
-      - okta.users.read
-      - okta.apps.read
-    privateKey: "~/.okta/privateKey.pem" # PEM format. This SDK supports RSA AND EC algorithms - RS256, RS384, RS512, ES256, ES384, ES512.
+    clientId: "yourClientId"
+    scopes: "okta.users.read okta.apps.read"
```

Comment:
Right now, multi line scopes is not working as expected. @arvindkrishnakumar-okta is aware of the issue. Once he fixes it, we can update the README if needed.

Reply:
@vijetmahabaleshwar-okta The issue you noted here was already noticed and fixed (see https://github.com/okta/okta-sdk-java/pull/379). Please build the latest SDK from master and inject it into your client app. You will not see this error.